### DISCUSSION TITLE
Is there a `transport-native-epoll` JAR that depends on both `linux-x86_64` and `linux-aarch_64`?

### DISCUSSION THREAD

**Discussion by hisener** [netty/netty#13758] [Q&A]
We depend on `transport-native-epoll`; however, we need to make sure to include platform-specific JARs, as well. Is there a Maven artifact that depends on all?

* `netty-transport-native-epoll:linux-aarch_64`
* `netty-transport-native-epoll:linux-x86_64`
* `netty-transport-native-kqueue:osx-aarch_64`
* `netty-transport-native-kqueue:osx-x86_64`

`netty-all` does this but it has more than these. 😅 

I am looking for a dependency similar to `netty-tcnative-boringssl-static` which depends on platform-specific JARs.

```
[INFO] +- io.netty:netty-tcnative-boringssl-static:jar:2.0.61.Final:compile
[INFO] |  +- io.netty:netty-tcnative-classes:jar:2.0.61.Final:compile
[INFO] |  +- io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.61.Final:compile
[INFO] |  +- io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.61.Final:compile
[INFO] |  +- io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.61.Final:compile
[INFO] |  +- io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.61.Final:compile
[INFO] |  \- io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.61.Final:compile
```

This is somewhat related to https://github.com/Azure/azure-sdk-for-java/discussions/38196.

**Comment by chrisvest** [comment]
No, besides `netty-all` we don't have a dependency that does that.

**Reply by hisener** [reply]
Does it make sense to have one similar to `netty-tcnative-boringssl-static`?
